src/cloudant/design_document.py

```diff
@@ -27,7 +27,7 @@ class DesignDocument(Document):
     :class:`~cloudant.document.Document`.  A DesignDocument object is
     instantiated with a reference to a database and
     provides an API to view management, list and show
```

Comment:
Also be sure that what you list here as potential upcoming design document API is accurate for CouchDB.  Can you confirm that show and list are valid for CouchDB?